Tests/_CryptoExtrasTests/TestRSABlindSigning.swift

```diff
@@ -0,0 +1,99 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCrypto open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftCrypto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import Foundation
+import XCTest
+import Crypto
+@testable import _CryptoExtras
+
+fileprivate struct TestVector: Codable {
+    var name, p, q, n, e, d, msg, msg_prefix, prepared_msg, salt, inv, blinded_msg, blind_sig, sig: String
+
+    var parameters: _RSA.BlindSigning.Parameters<SHA384> {
+        .init(
+            padding: self.salt.count == 0 ? .PSSZERO : .PSS,
+            preparation: self.msg == self.prepared_msg ? .identity : .randomized
+        )
+    }
+
+    static func load(from fileURL: URL) throws -> [Self] {
+        let json = try Data(contentsOf: fileURL)
+        let decoder = JSONDecoder()
+        return try decoder.decode([Self].self, from: json)
+    }
+}
+
+final class TestRSABlindSigning: XCTestCase {
+    func testAgainstRFC9474TestVectors() throws {
```

Comment:
Can we add test cases that ensure that we can go through the whole flow? 

In the case of test vectors, we skip the blinding step, so it would be good to verify that if we invoke all methods, we still have correctness. Would be also interesting to generate non-byte aligned key-sizes since for RSA those are possible.

Reply:
We cannot in this PR in its current form since we do not have any of the client-side functions implemented. This PR adds only blindSign and verify.

Reply:
Now we've included the client operations in this PR, we've been able to add full end-to-end flow tests for all named variants and supported key sizes.